website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -127,7 +127,7 @@ $ curl \
     http://127.0.0.1:8200/v1/sys/sync/destinations/aws-sm/my-store-1
 ```
 
-## Create|Update AWS Secrets Manager destination
+## Create/Update AWS Secrets Manager destination
```

Comment:
we uses `/` everywhere [else](https://developer.hashicorp.com/vault/api-docs/secret/kv/kv-v2#create-update-secret), while I was at it decided to homogenize